android/app/src/main/res/layout/activity_main.xml

```diff
@@ -7,48 +7,66 @@
 	<TextView
 		android:layout_width="match_parent"
 		android:layout_height="wrap_content"
-		android:text="Log In"
+		android:text="@string/log_in"
 		android:textAlignment="center"
 		android:layout_marginTop="30dp"
 		android:textSize="40sp"/>
 
 	<TextView
 		android:layout_width="match_parent"
 		android:layout_height="wrap_content"
-		android:layout_marginHorizontal="10dp"
-		android:text="Email:"
+		android:layout_marginStart="10dp"
+		android:text="@string/emailPrompt"
 		android:textSize="20sp" />
 
 	<EditText
 		android:layout_width="match_parent"
 		android:layout_height="wrap_content"
 		android:inputType="textEmailAddress"
-		android:layout_marginHorizontal="10dp"
+		android:layout_marginStart="10dp"
 		android:textSize="20sp"
 		android:id="@+id/username"
 		/>
 
 	<TextView
 		android:layout_width="match_parent"
 		android:layout_height="wrap_content"
-		android:layout_marginHorizontal="10dp"
-		android:text="Password:"
+		android:layout_marginStart="10dp"
+		android:text="@string/passwordPrompt"
 		android:textSize="20sp" />
 
 	<EditText
 		android:layout_width="match_parent"
 		android:layout_height="wrap_content"
 		android:inputType="textPassword"
-		android:layout_marginHorizontal="10dp"
+		android:layout_marginStart="10dp"
 		android:textSize="20sp"
 		android:id="@+id/password"
 		/>
 
-	<Button
-		android:layout_width="wrap_content"
+	<RelativeLayout
+		android:layout_width="match_parent"
+		android:layout_height="wrap_content">
+
+        <Button
+			android:layout_width="wrap_content"
+			android:layout_height="wrap_content"
+			android:text="@string/log_in"
+            android:layout_centerInParent="true"
+			android:id="@+id/loginButton"
+		/>
+
+	</RelativeLayout>
+
+	<TextView

```

Comment:
I was thinking maybe just a simple button that says "Register" or "Create Account".

Reply:
Like the Roastr iOS setup?

Reply:
Not sure. But it seems too wordy right now and could be reduced. The button would be a more apt view then.